configure

```diff
@@ -6691,6 +6696,15 @@ dag)
 	xxx_only=yes
 	;;
 
+dpdk)
+	#
+	# --with-pcap=dpdk is the only way to get here, and it means
+	# "DAG support but nothing else"
```

Comment:
You mean `"DPDK" support but nothing else"` here.

Reply:
But it's correct in `configure.ac`; perhaps you just need to run `autoreconf` and check in the result.

Reply:
I've done that.

Also, pcap-dpdk.c and pcap-dpdk.h need to be added to EXTRA_DIST, but I'll handle that as well.

Reply:
Cool, Thank you so much for the help and the code review.

Reply:
> Cool, Thank you so much for the help and the code review.

And thank you for the code!